packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -180,7 +223,7 @@ export default function LabeledField(props: Props) {
                             ? styles.labelWithDescription
                             : styles.labelWithNoDescription,
                         stylesProp?.label,
-                        hasError ? styles.labelWithError : undefined,
```

Comment:
Simplifying and consolidating some styles. I'll do another pass of consolidating and clean up in one of the next PRs once everything is place!